perses/annihilation/lambda_protocol.py

```diff
@@ -302,3 +362,56 @@ def set_alchemical_parameters(self,
        for parameter_name in lambda_protocol.functions:
            lambda_value = lambda_protocol.functions[parameter_name](beta0, beta)
            setattr(self, parameter_name, lambda_value)
+
+class RESTCapableRelativeAlchemicalState(AlchemicalState):
```

Comment:
Could we document (in a docstring) the meaning of all of the `lambda_*` parameters?